### PR TITLE
Moving init option from kpm

### PR DESCRIPTION
**Pull request by liangyuanpeng**
Releated: #57

#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [ ] Y 

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [ ] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [ ] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

### CODE REVIEW

File: cmd/kcl/commands/mod_init.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func NewModInitCmd(cli *client.KpmClient) *cobra.Command {
 				}
 			}
 
-			initOpts := opt.InitOptions{
+			initOpts := options.ModInitOptions{
```
[the remark this answers is not in the document]
Comment (Peefy) [reply cmd/kcl/commands/mod_init.go:57]: Just move here. ❤

**Comment by Peefy** [comment]
LGTM

**Comment by Peefy** [comment]
We will only abandon the part of KPM CLI and use kcl mod instead, while retaining all parts of kpm pkg.

**Comment by liangyuanpeng** [comment]
> Currently, we can write the option in the kpm repository and reference it in the cli repository.

Thanks for quickly reply and let me create a KPM pr for it,closing..

**Comment by Peefy** [comment]
Thank you for the contribution.

**Pull request closed without merging**